File: rb/test/exists_test.rb
# GlaxWeather SDK exists test

require "minitest/autorun"
require_relative "../GlaxWeather_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = GlaxWeatherSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- GlaxWeather SDK exists test

local sdk = require("glax-weather_sdk")

describe("GlaxWeatherSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
